config/settings.yml

```diff
@@ -248,3 +248,5 @@ terms:
 - term: Summer 2027
   quarter: Summer
   end_date: 2027-08-15
+
+certs_path: "../../shared/config/"
```

Comment:
In the app, this gets linked to `Rails.root.join('config')`. There shouldn't be a need to make this a special setting.